Common/src/main/resources/assets/snifferplus/lang/zh_cn.json

```diff
@@ -1,30 +1,30 @@
 {
-  "block.snifferplus.stone_pine_button": "五针松木按钮",
-  "block.snifferplus.stone_pine_door": "五针松木门",
-  "block.snifferplus.stone_pine_fence": "五针松木栅栏",
-  "block.snifferplus.stone_pine_fence_gate": "五针松木栅栏门",
-  "block.snifferplus.stone_pine_hanging_sign": "悬挂式五针松木告示牌",
-  "block.snifferplus.stone_pine_leaves": "五针松树叶",
-  "block.snifferplus.stone_pine_log": "五针松原木",
-  "block.snifferplus.stone_pine_planks": "五针松木板",
-  "block.snifferplus.stone_pine_pressure_plate": "五针松木压力板",
-  "block.snifferplus.stone_pine_sapling": "五针松树苗",
-  "block.snifferplus.stone_pine_sign": "五针松木告示牌",
-  "block.snifferplus.stone_pine_slab": "五针松木台阶",
-  "block.snifferplus.stone_pine_stairs": "五针松木楼梯",
-  "block.snifferplus.stone_pine_trapdoor": "五针松木活板门",
-  "block.snifferplus.stone_pine_wood": "五针松木",
-  "block.snifferplus.stripped_stone_pine_log": "去皮五针松原木",
-  "block.snifferplus.stripped_stone_pine_wood": "去皮五针松木",
+  "block.snifferplus.stone_pine_button": "笠松木按钮",
```

Comment:
The name changed for make the one more similar to the one in reality, it's same to below